src/osip-simulation-view/src/main/java/edu/kit/pse/osip/simulation/view/dialogs/HelpDialog.java

```diff
@@ -1,7 +1,146 @@
+
+        Button close = new Button(Translator.getInstance().getString("monitoring.helpdialog.closeButton"));
+        close.setDefaultButton(true);
+        close.setCancelButton(true);
+        close.setOnAction(new EventHandler<ActionEvent>() {
+            @Override
+            public void handle(ActionEvent actionEvent) {
+                Stage stage = (Stage) close.getScene().getWindow();
+                stage.close();
+            }
+        });
+
+        BorderPane pane = new BorderPane();
+        pane.setCenter(tabPane);
+        pane.setBottom(close);
+        pane.setAlignment(close, Pos.CENTER);
+        setScene(new Scene(pane));
+        setHeight(WINDOW_HEIGHT);
+        setWidth(WINDOW_WIDTH);
+    }
+
+    private Tab createMainTab() {
+        Tab tab = new Tab();
+        tab.setText(Translator.getInstance().getString("controller.helpdialog.mainTab.header"));
+
+        TextFlow textFlow = new TextFlow();
+
+        textFlow.getChildren().add(getText("controller.helpdialog.mainTab.introduction", null));
+        textFlow.getChildren().add(newLine());
+        textFlow.getChildren().add(getText("controller.helpdialog.mainTab.layoutSection", sectionFont));
+        textFlow.getChildren().add(getText("controller.helpdialog.mainTab.layoutText", null));
+
+        ScrollPane scrollPane = new ScrollPane();
+        scrollPane.setFitToWidth(true);
+        scrollPane.setContent(textFlow);
+        scrollPane.setPrefViewportHeight(textFlow.prefHeight(-1));
+        scrollPane.setPrefViewportWidth(textFlow.prefWidth(-1));
+        scrollPane.setPadding(new Insets(5));
+
+
+        tab.setContent(scrollPane);
+        tab.setClosable(false);
+
+        return tab;
+    }
+
+    private Text newLine() {
+        return new Text("\n");
+    }
+
+    private Text getText(String id, Font font) {
+        Text text = new Text();
+        text.setText(Translator.getInstance().getString(id) + "\n");
+        if (font != null) {
+            text.setFont(font);
+        }
+        return text;
+    }
+
+    private Tab createControlTab() {
+        Tab tab = new Tab();
+        tab.setText(Translator.getInstance().getString("controller.helpdialog.controlTab.header"));
+
+        TextFlow textFlow = new TextFlow();
+
+        textFlow.getChildren().add(getText("controller.helpdialog.controlTab.introduction", null));
+        textFlow.getChildren().add(newLine());
+        textFlow.getChildren().add(getText("controller.helpdialog.controlTab.generalSection", sectionFont));
+        textFlow.getChildren().add(getText("controller.helpdialog.controlTab.generalText", null));
+        textFlow.getChildren().add(newLine());
+        textFlow.getChildren().add(getText("controller.helpdialog.controlTab.mixTankSection", sectionFont));
+        textFlow.getChildren().add(getText("controller.helpdialog.controlTab.mixTankText", null));
+
+        ScrollPane scrollPane = new ScrollPane();
```

Comment:
Lohnt es sich vielleicht, die Erstellung der ScrollPanes in eine Methode auszulagern?